### PR TITLE
Sdks 1036 e2e test failure

### PR DESCRIPTION
**Pull request by jhoongo**

### CODE REVIEW

**Review by rodrigoareis — APPROVED** [review]
Changes looks good to me

File: FRTestHost/FRTestHost/SharedTestFiles/TestUtil/FRTestNetworkStubProtocol.swift [hunk 1]
```diff
@@ -36,6 +36,7 @@ class FRTestNetworkStubProtocol: URLProtocol {
     }
     
     public override class func canInit(with request: URLRequest) -> Bool {
+        FRTestNetworkStubProtocol.requestHistory.append(request)
```
Comment (rodrigoareis) [inline FRTestHost/FRTestHost/SharedTestFiles/TestUtil/FRTestNetworkStubProtocol.swift:39]: Missing year copyright update

File: FRTestHost/FRTestHost/SharedTestFiles/TestUtil/FRTestStubResponseParser.swift [hunk 2]
```diff
@@ -20,7 +20,7 @@ class FRTestStubResponseParser: NSObject {
     @objc public var error: Error?
     @objc public var redirectRequest: URLRequest?
     
-    @objc init? ( _ fileName: String) {
+    @objc init? ( _ fileName: String, _ baseUrl: String? = nil) {
```
Comment (rodrigoareis) [inline FRTestHost/FRTestHost/SharedTestFiles/TestUtil/FRTestStubResponseParser.swift:23]: Missing year copyright update

File: FRTestHost/FRTestHost/SharedTestFiles/TestUtil/FRTestUtils.swift [hunk 3]
```diff
@@ -41,10 +41,10 @@ class FRTestUtils: XCTest {
     }
     
     
-    @objc static func loadMockResponses(_ responseFileNames: [String]) {
+    @objc static func loadMockResponses(_ responseFileNames: [String], _ config: Config? = nil) {
```
Comment (rodrigoareis) [inline FRTestHost/FRTestHost/SharedTestFiles/TestUtil/FRTestUtils.swift:44]: Missing year copyright update

**Review by rodrigoareis — CHANGES_REQUESTED** [review]
Update the copyright header for the files with comments

**Review by rodrigoareis — APPROVED** [review]
Looks good to me

**Review by spetrov — APPROVED** [review]
Looks good to me! 👍🏻

**Pull request merged**